go.mod

```diff
@@ -73,6 +73,7 @@ require (
 	github.com/kisielk/errcheck v1.7.0
 	github.com/klauspost/compress v1.17.9
 	github.com/ks3sdklib/aws-sdk-go v1.2.9
+	github.com/lance6716/pebble v0.0.0-20241104073946-6f55c09bd183
```

Comment:
which in specific are changed in this fork?

Reply:
many SST user-defined metadata (called SST properties) adjustments to align with RocksDB 😂 After the whole feature is proved to work, I'll move this package to a better place